bucket/windows-terminal.json

```diff
@@ -0,0 +1,45 @@
+{
+    "homepage": "https://github.com/microsoft/terminal",
+    "description": "The new Windows Terminal, and the original Windows console host - all in the same place!",
+    "version": "0.6.2951.0",
+    "license": "MIT",
+    "url": "https://github.com/microsoft/terminal/releases/download/v0.6.2951.0/Microsoft.WindowsTerminal_0.6.2951.0_8wekyb3d8bbwe.msixbundle#/dl.7z",
+    "hash": "1e55cd46043ad5613242aae3bceb528b81a181a8b388015ebe7767000102d06a",
+    "architecture": {
+        "64bit": {
+            "pre_install": "gci $dir -Exclude CascadiaPackage_$($version)_x64.msix | rm -Recurse -Force"
+        },
+        "32bit": {
+            "pre_install": "gci $dir -Exclude CascadiaPackage_$($version)_x86.msix | rm -Recurse -Force"
+        }
+    },
+    "installer": {
```

Comment:
- Throw looks awefull and is not user friendly
- Major version and build number are mutually exclusive
	- If version is lower than 10 it will fail
	- Builds are increased over all windows versions so it is not possible to have higher builds on Lower windows versions

![A](https://i.imgur.com/whjBos3.png)
```suggestion
    "installer": {
        "script": [
            "$winVer = [Environment]::OSVersion.Version",
            "if (($winver.Major -lt '10') -or ($winVer.Build -lt 18362)) { error 'At least Windows 10 18362 is required.'; break }",
            "Get-ChildItem \"$dir\" '*.msix' | Select-Object -ExpandProperty Fullname | Expand-7zipArchive -DestinationPath \"$dir\" -Removal"
        ]
    },
```

See mocked version with hardcoded higher build version

![B](https://i.imgur.com/TizNz1C.png)

![OK](https://i.imgur.com/QagmPk1.png)